mirroring/mirroring.go

```diff
@@ -90,6 +91,42 @@ func GetLogInfo() (*models.LogInfo, error) {
 	return logInfoResp.GetPayload(), nil
 }
 
+func GetLogProof(firstSize *int64) (*models.ConsistencyProof, error) {
+	rekorServerURL := viper.GetString("rekorServerURL")
+	rekorClient, err := client.GetRekorClient(rekorServerURL)
```

Comment:
Hmm, I wonder if we'd like to split out the CLI/config setting parse routines from this function and make the server url a parameter instead...

Reply:
Yes, I had somewhat related doubt which is the fact that we are initializing a new rekor client for each operation we perform. We can make the server URL as a parameter but what do you think of making a rekor client instance as a parameter?

Reply:
Hmm, good point. Let's do the latter :+1: